spec/models/applicant_tracking_system_spec.rb

```diff
@@ -188,7 +188,7 @@
         title = feed.dig('jobs', 0, 'title')
         job_id = feed.dig('jobs', 0, 'id')
         company = CompanyCreator.call(ats: @ashbyhq, ats_identifier: 'lightdash')
-        job = JobCreator.call(ats: @ashbyhq, company:, job_id:)
+        job = JobCreator.call(ats: @ashbyhq, company:, job_id:, data: feed)
```

Comment:
Why do we have a data: feed now?

Reply:
I think I need to refactor that part at some point.
It seems useless if you dive into the JobCreator but if we don't need data (because we are going to fetch it later) why do we create feed in the first place?

Reply:
Sorry I'm confused - didn't you add it? What is it doing?

Reply:
I added it. But i did because of confusion. I remove it